admin_guide/out_of_resource_handling.adoc

```diff
@@ -60,6 +59,29 @@ When the node fails a pod, it terminates all containers in the pod, and
 the link:https://kubernetes.io/docs/concepts/workloads/pods/pod-lifecycle/#pod-phase[`PodPhase`] is transitioned to *Failed*.
 ====
 
+The node supports the `nodefs` and `imagefs` file system partitions when detecting disk pressure, as follows:
+
+* The `nodefs`, or `rootfs`, file system that the node uses for local disk volumes, daemon logs, emptyDir,
+and so on (for example, the file system that provides `/`). 
+* The `imagefs` file system that the container runtime uses for storing images and
+individual container-writable layers. Eviction thresholds are at 85% full for `imagefs`. The `imagefs` file system depends on the runtime and, in the case of Docker, which storage driver you are using:
++
+** For Docker, if you are using the `devicemapper` storage driver, the `imagefs` is thin pool. If you are using the `overlay2` storage driver, the `imagefs` is the file system that contains `/var/lib/docker/overlay2`.  
```

Comment:
I think this li needs to be two lis. (I also wonder if you need separate lis for \`imagefs\` and \`imagefs\` with Docker in the first-level ul.)

s/are using/use

Reply:
@kalexand-rh How does it look now?